pages/how-to-contribute.md

```diff
@@ -0,0 +1,149 @@
+| ------------------------------- | -------- | -----------
+| [Accumulo][a]                   | [C][ac]  | Core Project
+| [Accumulo Website][w]           | [C][wc]  | Source for this website
+| [Accumulo Examples][e]          | [C][ec]  | Accumulo example code
+| [Accumulo Testing][t]           | [C][tc]  | Accumulo test suites such as continuous ingest and random walk
+| [Accumulo Docker][d]            | [C][dc]  | Source for Accumulo Docker image
+| [Accumulo Wikisearch][s]        | [C][sc]  | Accumulo example application that indexes and queries Wikipedia data
+
+## Contribution workflow
+
+1. Create an [Apache JIRA account][jira-signup] (for issue tracking) and [GitHub account][github-join] (for pull requests).
+1. Find an [issue][newbie-issues] to work on or create one that describes the work that you want to do.
+1. [Fork] and [clone] the GitHub repository that you want to contribute to.
+1. Create a branch in the local clone of your fork.
+```    
+    git checkout -b accumulo-4321
+```    
+1. Do work and commit to your branch. You can reference [this link][messages] for a guide on 
+to write good commit log messages in git.
+1. Ensure you works satisfies the guidelines laid out in the `CONTRIBUTING.md` file.
+1. If needed, squash to the minimum number of commits. For help on squashing commits, see this tutorial or StackOverflow answer.
+1. [Push] your branch to your fork.
+```
+    git push origin accumulo-4321
+```
+1. Create a [Pull Request] on GitHub to the appropriate repository. If the work is not complete and the Pull Request is for feedback, please put `[WIP]` in the subject.
+1. At least one committer (and others in the community) will review your pull request and add any comments to your code.
+1. Push any changes from the review to the branch as new commits so the reviewer only needs to review new changes. Please avoid squashing commits after the review starts. Squashing makes it hard for the reviewer to follow the changes.
+1. Repeat this process until a reviewer approves the pull request.
+1. When the review process is finished, all commits on the pull request may be squashed by a committer. Please avoid squashing as it makes it difficult for the committer to know if they are merging what was reviewed.
+
+## Coding Guidelines
+
+* If a change needs to go into multiple branches of Accumulo, it should be merged into earlier branches then later branches. 
+* Accumulo follows [semver] for its public API. Accumulo lists which packages are public API in its [README.md][accumulo-readme]. 
+* Every file requires the ASF license header as described in [ASF Source Header][srcheaders].
+* Remove all trailing whitespaces. Eclipse users can use Source&rarr;Cleanup option to accomplish this.
+* Use 2 space indents and never use tabs!
+* Use 160-column line width for Java code and Javadoc.
+* Use a new line with single statement if/else blocks.
+* Do not use Author Tags. The code is developed and owned by the community.
```

Comment:
+1 for these Workflow and guidelines sections.  They are exactly what we need, concise directions.  These 2 sections alone could eliminate a lot of the wordy sections in the contributer-guide.